### PR TITLE
feat: Remove component-test from tar files

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.307.4064290734.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.307.4064290734.xml)  
[color_metro_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.307.4064290734.swift)  
[color_metro_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.307.4064290734.kt)  
[color_metro_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.307.4064290734.ts)  
[color_sbermarket_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.307.4064290734.swift)  
[color_sbermarket_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.307.4064290734.kt)  
[color_sbermarket_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.307.4064290734.ts)  
[color_sberprime_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.307.4064290734.swift)  
[color_sberprime_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.307.4064290734.kt)  
[color_sberprime_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.307.4064290734.ts)  
[color_selgros_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.307.4064290734.swift)  
[color_selgros_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.307.4064290734.kt)  
[color_selgros_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.307.4064290734.ts)  
[color_smbusiness_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.307.4064290734.swift)  
[color_smbusiness_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.307.4064290734.kt)  
[color_smbusiness_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.307.4064290734.ts)  
[PlasmaTokensColor--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.307.4064290734.swift)  
[shadow_sbermarket_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.307.4064290734.ts)  
[typo_mage_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.307.4064290734.swift)  
[typo_mage_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.307.4064290734.kt)  
[typo_mage_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.307.4064290734.ts)  
[typo_plasma_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.307.4064290734.swift)  
[typo_plasma_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.307.4064290734.kt)  
[typo_plasma_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.307.4064290734.ts)  
[typo_ruler_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.307.4064290734.swift)  
[typo_ruler_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.307.4064290734.kt)  
[typo_ruler_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.307.4064290734.ts)  
[typo_sage_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.307.4064290734.swift)  
[typo_sage_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.307.4064290734.kt)  
[typo_sage_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.307.4064290734.ts)  
[typo_sbermarket_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.307.4064290734.swift)  
[typo_sbermarket_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.307.4064290734.kt)  
[typo_sbermarket_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.307.4064290734.ts)  
[typo_soulmate_ios-swift--canary.307.4064290734.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.307.4064290734.swift)  
[typo_soulmate_kotlin--canary.307.4064290734.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.307.4064290734.kt)  
[typo_soulmate_react-native--canary.307.4064290734.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.307.4064290734.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.131.0-canary.307.4064290734.0
  npm install @salutejs/plasma-temple@1.128.0-canary.307.4064290734.0
  npm install @salutejs/plasma-ui@1.161.0-canary.307.4064290734.0
  npm install @salutejs/plasma-web@1.158.0-canary.307.4064290734.0
  # or 
  yarn add @salutejs/plasma-b2c@1.131.0-canary.307.4064290734.0
  yarn add @salutejs/plasma-temple@1.128.0-canary.307.4064290734.0
  yarn add @salutejs/plasma-ui@1.161.0-canary.307.4064290734.0
  yarn add @salutejs/plasma-web@1.158.0-canary.307.4064290734.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-307/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-307/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-307/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-307/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-307/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-307/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-307/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-307/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-307/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-307/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-307/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-307/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-307/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-307/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-307/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-307/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-307/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-307/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-307/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-307/temple-storybook/

File: .babelrc.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const plugins = [
 ];
 const ignore = ['**/*.d.ts', '**/helpers', '**/__helpers', '**/*.stories.tsx'];
 
+!process.env.INCLUDE_TEST_FILES && ignore.push('**/*.component-test.tsx');
```
Comment (Yakutoc) [inline .babelrc.js:17]: @neretin-trike объясни пожалуйста как это работает? )

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-307/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-307/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-307/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-307/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

File: .babelrc.js (hunk 1 above)
Comment (neretin-trike) [reply .babelrc.js:17]: для случаев, когда INCLUDE_TEST_FILES = true, мы убираем из игнора маску тестов, и таким образом они попадут в тарник. Иначе (по умолчанию) сборка будет происходить без файлов тестов

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-307/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-307/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-307/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-307/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-307/temple-storybook/

**Review by Yeti-or — APPROVED** [review]
Сработало: 
```
 % du -h plasma-ui-1.161.0*
320K	plasma-ui-1.161.0-canary.307.4064290734.0.tgz
448K	plasma-ui-1.161.0.tgz
```

**Pull request merged**